apps/dataflow/src/scraper/fms/get_all_svg.py

```
@@ -0,0 +1,241 @@
+    with p.open("r", encoding="utf-8") as f:
+        data = json.load(f)
+    if not isinstance(data, list):
+        raise ValueError("buildings.json must be a JSON list of objects.")
+    for obj in data:
+        if not isinstance(obj, dict) or "building" not in obj or "floorid" not in obj:
+            raise ValueError("Each item must be an object with 'building' and 'floorid'.")
+        if not isinstance(obj["floorid"], dict):
+            raise ValueError("'floorid' must be an object mapping floor labels to floor IDs.")
+    return data
+
+def fetch_svg(
```

Ruff (0.14.14)

[warning] Too many arguments in function definition (9 > 5)

(PLR0913)